pr-data.csv

```diff
@@ -809,10 +809,10 @@ https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop
 https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-hdfs-project/hadoop-hdfs-nfs,org.apache.hadoop.hdfs.nfs.nfs3.TestWrites.testOverlappingWrites,OD-Vic,,,
 https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-hdfs-project/hadoop-hdfs-nfs,org.apache.hadoop.hdfs.nfs.nfs3.TestWrites.testWriteStableHow,OD-Vic,,,
 https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.hdfs.TestNNBench.testNNBenchCreateReadAndDelete,ID,,,
-https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.oauth2.TestClientCredentialTimeBasedTokenRefresher.refreshUrlIsCorrect,ID,,,
-https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.oauth2.TestRefreshTokenTimeBasedTokenRefresher.refreshUrlIsCorrect,ID,,,
+https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.oauth2.TestClientCredentialTimeBasedTokenRefresher.refreshUrlIsCorrect,ID,DeveloperFixed,,
+https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.oauth2.TestRefreshTokenTimeBasedTokenRefresher.refreshUrlIsCorrect,ID,DeveloperFixed,,
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.TestURLConnectionFactory.testSSLFactoryCleanup,OD,,,
-https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-common-project/hadoop-common,org.apache.hadoop.ipc.TestDecayRpcScheduler.testPriority,ID,,,
+https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-common-project/hadoop-common,org.apache.hadoop.ipc.TestDecayRpcScheduler.testPriority,ID,Rejected,https://github.com/apache/hadoop/pull/5085,
```

Comment:
Why is this rejected? The PR is still open and it needs some followup.

Reply:
Indeed, there was no follow up for 3 weeks.

Reply:
And I'm not quite sure about the answer of the question raised by the developer that in which case/scenario the test will fail.

Reply:
What is the root cause of the failure? Does it fail because of (1) the iteration order of `HashMap`, (2) `getDeclaredFields`, or (3) something third?

Reply:
It's caused by the process of converting an object to a string. The attributes of that object were converted into a non-deterministic order.